### ISSUE TITLE
New record based from object literal ? 

### ISSUE THREAD

**Issue by andrzejpakula** [typeorm/typeorm#2487]
[x ] question

Hi. Is it a normal behavior that the save() function in the repository creates a new record in the database based on the object literal instead of the entity instance without the previously used preload() function ?

**Comment by pleerock** [comment]
yes it is. You can send partial objects to save method and it will only update what you sent.
